ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/impl/PlatformFileInfoService.java

```diff
@@ -55,4 +55,12 @@ public String getFileContentById(Integer fileId) {
     }
   }
 
+  public byte[] getFileContentBytesById(Integer fileId) {
```

Comment:
Is the method `getFileContentById ` still needed?

Reply:
There is no call of the method getFileContentById left. 
 i will remove it.